docs/documentation/exif.rst

```diff
@@ -0,0 +1,62 @@
+Exif
+====
+
+Vimiv provides optional exif support if either `pyexiv2`_ or `piexif`_ is available. If
+this is the case:
+
+#. Exif metadata is automatically copied from source to destination when writing images
+   to disk.
+#. The ``:metadata`` command and the corresponding ``i``-keybinding is available.
+#. The ``{exif-date-time}`` statusbar module is available.
+
+
+Advantages of the different exif libraries
+------------------------------------------
+
+`Pyexiv2`_ is the more powerful of the two options. One large advantage is that it has
```

Comment:
I would focus more on the difference it makes on the user: _One large advantage of pyexiv2 is that `:metadata` formats the exif data into human readable format, for example ``FocalLength: 5.0 mm`` where `piexif`_ would only give ``FocalLength: 5.0``._